gnomad/variant_qc/pipeline.py

```diff
@@ -276,7 +278,6 @@ def generate_trio_stats(mt: hl.MatrixTable,) -> hl.Table:
 def generate_sib_stats(
     mt: hl.MatrixTable,
     relatedness_ht: hl.Table,
-    sex_ht: hl.Table,
```

Comment:
are you planning to add this back in the future?

Reply:
i don't think we will need it if we are filtering to autosomes now

Reply:
yup that makes sense, but generate trio stats expr still allows for sex chromosomes. maybe we should change that as well to be consistent?

Reply:
Ah, so just remove `proband_is_female_expr=mt.is_female,` in `generate_trio_stats`

Reply:
or do you think it would be better to require sibling stats to have is_female?

Reply:
maybe the first one (removing `is_female`). that way, both `generate_trio_stats` and `generate_sib_stats` will default to computing on autosomes only, but users can run `generate_trio_stats_expr` or `generate_sib_stats_expr` if they want sex chromosomes for whatever reason